rfq/manager.go

```diff
@@ -112,6 +113,8 @@ type ManagerCfg struct {
 	// into the manager once lnd and tapd are hooked together.
 	AliasManager ScidAliasManager
 
+	AuxChanNegotiator *tapfeatures.AuxChannelNegotiator
```

Comment:
See my other comment on rfq/order.go, which I wrote first, but it would be nice if we could avoid sticking this in here, propagating only a single 'UseNoOptHTLCs' flag. Is it possible to resolve that stuff any earlier?